lib/src/rx_command_builder.dart

```diff
@@ -4,20 +4,20 @@ import 'package:flutter/material.dart';
 import 'package:rx_command/rx_command.dart';
 import 'package:rx_widgets/src/builder_functions.dart';
 
-/// Spinner/Busy indicator that reacts on the output of a `Stream<CommandResult<T>>`.
+/// Spinner/Busy indicator that reacts on the output of a `Stream<CommandResult<dynamic, R>>`.
 /// It's made especially to work together with `RxCommand` from the `rx_command`package.
 /// it starts running as soon as an item with `isExecuting==true` is received  until `isExecuting==true` is received.
 /// To react on other possible states (`data, no data, error`) that can be emitted it offers three option `Builder` methods.
-class RxCommandBuilder<T> extends StatelessWidget {
-  final Stream<CommandResult<T>> commandResults;
-  final RxBuilder<T> dataBuilder;
-  final ErrorBuilder<Exception> errorBuilder;
-  final BusyBuilder busyBuilder;
-  final PlaceHolderBuilder placeHolderBuilder;
-  final TargetPlatform platform;
+class RxCommandBuilder<R> extends StatelessWidget {
+  final Stream<CommandResult<dynamic, R>> commandResults;
+  final RxBuilder<R>? dataBuilder;
+  final ErrorBuilder<Exception>? errorBuilder;
```

Comment:
Wouldn't it make sense to pass the full `CommandError` to the `errorBuilder` ? On that note I just wonder if not all error handlers/builders should be defined to expect `CommandErrors`. 
I know this would be a major breaking change, but the ability to access the `paramData` is a huge advantage to display meaningful error messages

Reply:
This makes sense, didn't see it before. I think will use it.

Reply:
Updated. I don't think we should define CommandErrors in all builders as this one is specific to rxCommand.

Reply:
Please review. all changes.